config_defaults_inc.php

```diff
@@ -2518,6 +2542,15 @@
 $g_report_bug_threshold = REPORTER;
 
 /**
+ * Enables / disables ability to create new bug report by posting to 
+ * bug_report.php with 'auth_username' and 'auth_password' fields set.
+ * Only API tokens are allowed.
+ * This can be used for automatic collection of bug reports.
+ * @global integer $g_bug_auto_reports_submission_enabled
+ */
+$g_bug_auto_reports_submission_enabled = OFF;
```

Comment:
This option controls only secondary entry point in bug_report.php.

I don't know if this option is really needed (I do not see usage cases for it), I have added it just in case.